sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_models.py

```diff
@@ -48,11 +48,70 @@ class SettingSelector:
     :keyword label_filter: A filter to select configuration settings based on their labels. Default is value is
      EMPTY_LABEL i.e. (No Label) as seen in the portal.
     :type label_filter: Optional[str]
+    :keyword tag_filters: A filter to select configuration settings based on their tags. This is a list of strings
```

Comment:
Is this common between FF select options and kv select?

Reply:
Yes. We do use the term configuration setting here and in the other ones. So it isn't super clear that it's for feature flags too.

Reply:
@avanigupta do you have a suggestion to update the text?

Reply:
How about - "A filter to select configuration settings and feature flags based on their tags."
This applies to all filter descriptions